### ISSUE TITLE
sshnp / npt '-q' option to prevent the "noise"

### ISSUE THREAD

**Issue by cconstab** [atsign-foundation/noports#1066]
### Is your feature request related to a problem? Please describe.

Sometimes it is nice to have the connection status as sshnp/npt make the connect but its pretty ugly using 2>/dev/null if you want to prevent it showing (for example when using sshnp/npt in a ~/.ssh/config file)

### Describe the solution you'd like

'-q` option that prevents the connection messages
i.e. these
```
2024-04-24 14:18:50.703794 : Sending daemon feature check request
2024-04-24 14:18:50.703855 : Resolving remote username for user session
2024-04-24 14:18:51.589465 : Resolving remote username for tunnel session
2024-04-24 14:18:51.589627 : Fetching host and port from srvd
2024-04-24 14:18:54.983525 : Received host and port from srvd
2024-04-24 14:18:54.983557 : Waiting for daemon feature check response
2024-04-24 14:18:54.983569 : Received daemon feature check response
2024-04-24 14:18:54.985528 : Required daemon features are supported
2024-04-24 14:18:55.140527 : Sending session request to the device daemon
2024-04-24 14:18:55.360552 : Waiting for response from the device daemon
2024-04-24 14:18:56.266542 : Received response from the device daemon
2024-04-24 14:18:56.266656 : Creating connection to socket rendezvous
2024-04-24 14:18:56.377485 : Starting tunnel session
2024-04-24 14:18:56.845509 : Starting user session
```

### Describe alternatives you've considered

using 2>/dev/null

**Comment by gkc** [comment]
This issue was created in the wrong repo; moving to noports repo

**Comment by gkc** [comment]
Closed by #1077
